app/presenters/school_presenter.rb

```diff
@@ -3,6 +3,20 @@ def location
     [model.name, model.town, model.county].reject(&:blank?).join(', ')
   end
 
+  def school_size
+    if model.gias_data.present?
+      return model.gias_data['NumberOfPupils'] + ' pupils enrolled' if model.gias_data['NumberOfPupils'].present?
+      return 'Up to ' + model.gias_data['SchoolCapacity'] + 'pupils' if model.gias_data['SchoolCapacity'].present?
+    end
+    'No data available'
+  end
+
+  def ofsted_report
+    if model.gias_data.present?
+      "https://www.ofsted.gov.uk/oxedu_providers/full/(urn)/#{model.gias_data['URN']}"
```

Comment:
Make the ofstead endpoint a frozen constant at the top of the presenter, please.  Makes is easier to find and change if we have to.

Reply:
Will do! Do you have a name you'd prefer? I'll probably go with `OFSTED_REPORT_ENDPOINT` unless you have a preference for a different name